art-community-platform/backend/app/api/views/searchUser.py

```diff
@@ -0,0 +1,59 @@
+from django.http import HttpResponse, HttpResponseBadRequest, JsonResponse
+from rest_framework.decorators import api_view
+import hashlib
+import json
+
+from ..helpers.comment_helpers import get_comment_by_id_helper
+from ..helpers.exhibition_helpers import get_exhibition_by_id_helper
+from ..helpers.notification_helpers import get_notification_by_id_helper
+from ..helpers.tag_helpers import get_tag_by_id_helper
+from ..helpers.art_item_helpers import *
+from ..helpers.user_helpers import *
+from ..models.user import User
+from ..models.art_item import ArtItem
+from ..models.comment import Comment
+from ..models.tag import Tag
+
+
+
+@api_view(['GET'])
+def searchUser(req,keyword):
+
+    userIDs = []
+    userList = User.objects.all()
```

Comment:
Instead of getting all objects, you can query them and only get the necessary search results. Note that, this suggestion doesn't affect correctness, it may affect performance
This version also works fine.